pkg/component/registrycaches/registry_caches.go

```diff
@@ -292,6 +294,22 @@ func (r *registryCaches) computeResourcesDataForRegistryCache(ctx context.Contex
 							Type: corev1.SeccompProfileTypeRuntimeDefault,
 						},
 					},
+					InitContainers: []corev1.Container{
+						{
+							// Mitigation for  https://github.com/distribution/distribution/issues/4478.
+							Name:            "cleanup-volume",
+							Image:           r.values.InitImage,
+							ImagePullPolicy: corev1.PullIfNotPresent,
+							Command:         []string{"sh", "-c", "if [ -f /var/lib/registry/scheduler-state.json ]; then if [ -s /var/lib/registry/scheduler-state.json ]; then echo 'scheduler-state.json is OK'; else echo 'cleanup corrupted scheduler-state.json'; rm -f /var/lib/registry/scheduler-state.json; echo 'clean up docker directory'; rm -rf /var/lib/registry/docker; fi; else echo 'scheduler-state.json is not created yet'; fi"},
```

Comment:
Can you write it as multiline string with the golang's multiline string syntax? It is not readable and maintainable written in a single line.

Example https://github.com/gardener/gardener/blob/1832cc548ca2cf8eb9bd2b2454728f217ec14f01/test/framework/resources/templates/logger-app.yaml.tpl#L24-L36